<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projetsaif</class>
 <widget class="QMainWindow" name="projetsaif">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>projetsaif</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des Agents</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajouter Agent</string>
       </attribute>
       <widget class="QPushButton" name="ajouterBusAnnuler">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>420</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbAgentAjout">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>420</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Confirmer</string>
        </property>
       </widget>
       <widget class="QLabel" name="LabelDestination">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Prenom d'Agent</string>
        </property>
       </widget>
       <widget class="QLabel" name="LabelImmatricuteBus">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>CIN d'agent</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LECINAgentAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEPrenomAgentAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>120</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="LENomAgentAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>70</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="LabelIDConducteur">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>L'id de service</string>
        </property>
       </widget>
       <widget class="QLabel" name="LabelNbPlace">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Nom d'Agent</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEIDServiceAgentAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>350</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Date de naissance</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Salaire</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEDateNaisAgentAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>180</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="LESalaireAgentAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>240</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="LESexeAgentAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>290</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Modifier Agent</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>811</width>
          <height>521</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_11">
         <attribute name="title">
          <string>Agent a modifier</string>
         </attribute>
         <widget class="QLabel" name="labelModifierBus">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>381</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Veuillez saisir le CIN de d'agent que vous souhaitez le modifier</string>
          </property>
         </widget>
         <widget class="QPushButton" name="ConfirmerModifBus1">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>150</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Confirmer</string>
          </property>
         </widget>
         <widget class="QPushButton" name="modifierBusRetourner">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>150</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Retourner</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="LEModifierBus">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>70</y>
            <width>181</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_12">
         <attribute name="title">
          <string>Modification du Agent</string>
         </attribute>
         <widget class="QLineEdit" name="LEIDServiceAgentModif2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>350</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="ajouterBusAnnuler_2">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>420</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
         <widget class="QLabel" name="LabelImmatricuteBus_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>CIN d'agent</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="LESalaireAgentModif2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>240</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="LabelIDConducteur_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>340</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>L'id de service</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="LEDateNaisAgentModif2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>180</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="LENomAgentModif2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>70</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Date de naissance</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pbAgentModif2">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>420</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Confirmer</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="LESexeAgentModif2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>290</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="LabelDestination_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Prenom d'Agent</string>
          </property>
         </widget>
         <widget class="QLabel" name="LabelNbPlace_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Nom d'Agent</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="LECINAgentModif2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>10</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>300</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Sexe</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Salaire</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="LEPrenomAgentModif2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>120</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Supprimer Agent</string>
       </attribute>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbAgentSupprimer">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LECINAgentSupprimer">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Afficher Agent</string>
       </attribute>
       <widget class="QTableView" name="tabAgent">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>641</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des Service</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajouter Service</string>
       </attribute>
       <widget class="QLabel" name="LabelImmatricuteBus_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>ID de service</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEIDServiceAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="LELibelleServiceAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>80</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="LabelNbPlace_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Libelle service</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbServiceAjouter">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Capacite</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LECapaciteServiceAjout">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>140</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Modifier Service</string>
       </attribute>
       <widget class="QLabel" name="LabelImmatricuteBus_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>ID de service</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LELibelleServiceModif2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>90</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>220</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEIDServiceModif2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="LECapaciteServiceModif2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>150</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Capacite</string>
        </property>
       </widget>
       <widget class="QLabel" name="LabelNbPlace_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Libelle service</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbServiceModif2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>220</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Supprimer Service</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>140</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbServiceSupprimer">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>140</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LEIDServiceSupprimer">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="LabelImmatricuteBus_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 8pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>ID de service</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Afficher Service</string>
       </attribute>
       <widget class="QTableView" name="tabService">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
